lib/generators/debugs_bunny/trace/trace_generator.rb

```diff
@@ -3,14 +3,15 @@
 require 'rails/generators'
 
 module DebugsBunny
-  class ModelGenerator < Rails::Generators::NamedBase
-    source_root File.expand_path('templates', __dir__)
+  class TraceGenerator < Rails::Generators::NamedBase
+    TEMPLATE_DIR = File.join(TEMPLATES_DIR, 'model')
```

Comment:
Should `model` be singular here?

Reply:
In its current state, yes; this is only because I named the internal template folder 'model'. For consistency, I could rename it to 'models' - I think that's a good idea.